dockerfiles/README.md

```diff
@@ -83,24 +83,79 @@
 ## OpenVINO Version (Preview)
 #### Linux 16.04, Python Bindings
 
-1. Build the docker image from the Dockerfile in this repository.
-  ```
-  # If you have a Linux machine, preface this command with "sudo"
-
-  docker build -t onnxruntime-openvino -f Dockerfile.openvino .
-  ```
-  To use GPU_FP32:
-  ```
-  docker build -t onnxruntime-openvino --build-arg TARGET_DEVICE=GPU_FP32 -f Dockerfile.openvino .
-  ```
-
-2. Run the Docker image
-
-  ```
-  # If you have a Linux machine, preface this command with "sudo"
-
-  docker run -it onnxruntime-openvino
-  ```
+1. Build the onnxruntime image for all the accelerators supported as below 
+
+   Retrieve your docker image in one of the following ways.
+
+    -  For building the docker image, download OpenVINO online installer version 2018 R5.0.1 from [here](https://software.intel.com/en-us/openvino-toolkit/choose-download) and copy the openvino tar file in the same directory and build the image. The online installer size is only 16MB and the components needed for the accelerators are mentioned in the dockerfile.
```

Comment:
I went to that link and couldn't download 2018 R5.0.1  , they only seem to have 2019 R1 available for download
Is there another direct link to R5? otherwise, this will be a problem.

Reply:
I found https://download.01.org/openvinotoolkit/2018_R5/packages/ 
Is it officially supported?

Reply:
the file I downloaded from there was l_openvino_toolkit_ie_p_2018.5.445.tgz
i don't think it works with the Dockerfile.openvino since it untars to a directory called inference_engine_vpu_arm and doesn't have some required files.

Reply:
Attaching the screenshot for downloading the R5 version 
![image](https://user-images.githubusercontent.com/41489828/61078739-26c0a900-a3d6-11e9-8957-002ce52c3a49.png)

Reply:
R5 is not currently an option. I just signed up this morning. See screenshot. 

![Screenshot from 2019-07-11 12-43-33](https://user-images.githubusercontent.com/43355415/61080419-de0aef00-a3d9-11e9-8ee6-960d29d49609.png)

Reply:
We are trying to resolve this issue. Will email as soon as it is fixed.

Reply:
any updates on this issue?